### ISSUE TITLE
Wireframe not transparent in 0.33.3

### ISSUE THREAD

**Issue by hakostra** [pyvista/pyvista#2353]
I updated to Pyvista 0.33.3 today, from 0.31.3. I still use the same VTK build.

I have the following small plotting script:

```
pv.set_plot_theme("document")
plotter = pv.Plotter(window_size=[1920, 1200])

plotter.add_mesh(grid, scalars='LEVEL', style="wireframe", categories=True)
_ = plotter.add_axes(line_width=5)

plotter.camera_position = 'iso'
plotter.show(screenshot='grids.png', auto_close=False)

plotter.close()
```
that previously (0.31.3) produced this plot:
![grids-0 31 3](https://user-images.githubusercontent.com/3898672/158613142-5cce3012-da70-49d8-8096-3a7f9ca3d6c3.png)

With 0.33.3 the result is:
![grids-0 33 3](https://user-images.githubusercontent.com/3898672/158614043-fc9d64d8-f398-4d57-99ca-c405aae63fd7.png)

I.e. the wireframe representation is no longer having transparent faces. I consider this to be slightly awkward and a bug.

I uploaded a complete reproducer script and the input file used here:
[wireframe.tar.gz](https://github.com/pyvista/pyvista/files/8263137/wireframe.tar.gz)

**System Information:**

0.31.3:
```txt
--------------------------------------------------------------------------------
  Date: Wed Mar 16 14:22:42 2022 UTC

                OS : Linux
            CPU(s) : 16
           Machine : x86_64
      Architecture : 64bit
               RAM : 62.6 GiB
       Environment : Python
        GPU Vendor : Mesa/X.org
      GPU Renderer : llvmpipe (LLVM 12.0.1, 256 bits)
       GPU Version : 4.5 (Core Profile) Mesa 21.3.0

  Python 3.8.11 (default, Sep  1 2021, 12:33:46)  [GCC 9.3.1 20200408 (Red Hat
  9.3.1-2)]

           pyvista : 0.31.3
               vtk : 9.1.20211118
             numpy : 1.21.4
           imageio : 2.13.3
           appdirs : 1.4.4
            scooby : 0.5.9
            meshio : 4.4.6
        matplotlib : 3.4.3
          colorcet : 3.0.0
             scipy : 1.7.3
--------------------------------------------------------------------------------
```

0.33.3:
```txt
--------------------------------------------------------------------------------
  Date: Wed Mar 16 14:22:23 2022 UTC

                OS : Linux
            CPU(s) : 16
           Machine : x86_64
      Architecture : 64bit
               RAM : 62.6 GiB
       Environment : Python
        GPU Vendor : Mesa/X.org
      GPU Renderer : llvmpipe (LLVM 12.0.1, 256 bits)
       GPU Version : 4.5 (Core Profile) Mesa 21.3.0

  Python 3.8.11 (default, Sep  1 2021, 12:33:46)  [GCC 9.3.1 20200408 (Red Hat
  9.3.1-2)]

           pyvista : 0.33.3
               vtk : 9.1.20211118
             numpy : 1.22.3
           imageio : 2.16.1
           appdirs : 1.4.4
            scooby : 0.5.12
        matplotlib : 3.4.3
          colorcet : 3.0.0
             scipy : 1.8.0
--------------------------------------------------------------------------------
```

**Comment by hakostra** [comment]
I believe the change stems from this commit: https://github.com/pyvista/pyvista/pull/1523/commits/95ccfc42f42659b7028e0ba27306772cb81003d6

With ``plotter.disable_hidden_line_removal()`` and parallel projection this is what I want to see:

![grids](https://user-images.githubusercontent.com/3898672/158761679-54ae265e-757a-49e1-847d-83d1bfd93bd7.png)

I have to say I completely disagree with enabling hidden line removal for a single theme only. It might be looking "nicer" in a specific case, howerver, it completely breaks the general wireframe representation. Even stranger is it that it does that only for a single theme. If there is consensus that wireframe (as Paraview shows it) is not wireframe in Pyvista, then at least it should be consistent and do it for all themes.

In my opinion, the current 0.33.3 behavior is a visualization with the faces of the cells painted in the same color as the background. I.e. the visualization style of "wireframe" in Pyvista is now that of "Surface with edges" in Paraview. This is extremely confusing.

**Comment by banesullivan** [comment]
> I have to say I completely disagree with enabling hidden line removal for a single theme only.

Good catch! and I agree with you. This will be reverted.

**Comment by akaszynski** [comment]
Agree with reverting this as well.

**Comment by hakostra** [comment]
This issue was resolved in the merged PR #2488 (why wasn't the issue closed automatically?)
